lib/build-query/builder.js

```diff
@@ -136,6 +138,7 @@ const buildItemEstimatesQuery = (doc) => ({
   'lux:itemEvents': eventsWithItem(doc.id),
   'lux:itemDepartment': departmentsWithItem(doc.id),
   'lux:itemUnit': itemById(doc.id),
+  'lux:itemSubjectOf': itemsSubjectOfItem(doc.id),
```

Comment:
This line should go into `buildWorkEstimatesQuery`, because `subjectOfItem` has a target scope of `work`